src/main/java/com/manufacturing/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,120 @@
+package com.manufacturing.dao.jdbc;
+
+import com.manufacturing.dao.ManufacturerDao;
+import com.manufacturing.lib.Dao;
+import com.manufacturing.model.Manufacturer;
+import com.manufacturing.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    private static final String MANUFACTURER_NAME = "manufacturer_name";
+    private static final String MANUFACTURER_ID = "manufacturer_id";
+    private static final String ORIGIN = "origin";
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturers (manufacturer_name, origin) VALUE (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(query,
+                    Statement.RETURN_GENERATED_KEYS);
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject(1, Long.class));
+            }
+            statement.close();
+            resultSet.close();
```

Comment:
it is optional to close ResultSet, because when a Statement object is closed, its current ResultSet object, if one exists, is also closed.
https://docs.oracle.com/en/java/javase/11/docs/api/java.sql/java/sql/Statement.html#close()